tests/cases/standalone/common/timestamp/timestamp.result

```diff
@@ -26,14 +26,57 @@ INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('2023-04-04 08:00:00.0052+0
 
 Affected Rows: 1
 
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('+100000-01-01 00:00:01.5Z', 3);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('-262144-01-01 00:00:00Z', 4);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('+262143-12-31 23:59:59Z', 5);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('-262144-01-01 00:00:00Z', 6);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('+262143-12-31 23:59:59.999Z', 7);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('-262144-01-01 00:00:00Z', 8);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('+262143-12-31 23:59:59.999999Z', 9);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('1677-09-21 00:12:43.145225Z', 10);
+
+Affected Rows: 1
+
+INSERT INTO timestamp_with_precision(ts,cnt) VALUES ('2262-04-11 23:47:16.854775807Z', 11);
+
+Affected Rows: 1
+
 SELECT * FROM timestamp_with_precision ORDER BY ts ASC;
 
-+----------------------------+-----+
-| ts                         | cnt |
-+----------------------------+-----+
-| 2023-04-04T00:00:00.005200 | 2   |
-| 2023-04-04T08:00:00.005200 | 1   |
-+----------------------------+-----+
++-------------------------------+-----+
+| ts                            | cnt |
++-------------------------------+-----+
+| -262144-01-01T00:00:00        | 8   |
+| 1677-09-21T00:12:43.145225    | 10  |
+| 2023-04-04T00:00:00.005200    | 2   |
+| 2023-04-04T08:00:00.005200    | 1   |
+| 2262-04-11T23:47:16.854775    | 11  |
+| +100000-01-01T00:00:01.500    | 3   |
+| +262143-12-31T23:59:59        | 5   |
+| +262143-12-31T23:59:59.999    | 7   |
+| +262143-12-31T23:59:59.999999 | 9   |
```

Comment:
Are those `+` redundant?

Reply:
> Are those `+` redundant?

Nope, chrono requires this `+` to parse very large timestamps that exceeds ISO-8601 timestamp range.

Reply:
🤣